assets/js/modules/search-console/index.js

```diff
@@ -1,7 +1,7 @@
 /**
- * Search Console module initialization.
+ * Search console module initialization.
```

Comment:
Nitpicking, but this should be capitalized since it's a product name :)

Reply:
Fixed